src/main/java/seedu/resireg/logic/parser/ReallocateCommandParser.java

```diff
@@ -0,0 +1,51 @@
+package seedu.resireg.logic.parser;
+
+import static seedu.resireg.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.resireg.logic.parser.CliSyntax.PREFIX_ROOM_INDEX;
+import static seedu.resireg.logic.parser.CliSyntax.PREFIX_STUDENT_INDEX;
+
+import java.util.stream.Stream;
+
+import seedu.resireg.commons.core.index.Index;
+import seedu.resireg.logic.commands.ReallocateCommand;
+import seedu.resireg.logic.parser.exceptions.ParseException;
+
+
+/**
+ * Parses input arguments and creates a new AddCommand object
+ */
+public class ReallocateCommandParser implements Parser<ReallocateCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand
```

Comment:
`AddCommand` should be replace by `ReallocateCommand`

Reply:
Fixed.